GCLM and EHD with distance function

AndyLeeZaiZai opened the pull request.

AndyLeeZaiZai committed:
GCLM and EHD with distance function

Samonum commented on app/features/EHD.py:
```diff
@@ -0,0 +1,50 @@
+from scipy.spatial import distance
+from SimpleCV import Image, np, EdgeHistogramFeatureExtractor
+
+def EHDFeatures(image):
```
What sort of image is image? OpenCV image?

AndyLeeZaiZai replied:
Opencv image works.

AndyLeeZaiZai committed:
Some updates

1. upadate GLCM with Bhattacharyya distance
2. update EHD with Bhattacharyya distance
3. add Gabor filter features, also with Bhattacharyya distance
